docs/programming-guide.md

```diff
@@ -24,7 +24,8 @@ along with if you launch Spark's interactive shell -- either `bin/spark-shell` f
 
 <div data-lang="scala"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} uses Scala {{site.SCALA_BINARY_VERSION}}. To write
+Spark {{site.SPARK_VERSION}} is built and distributed to work with Scala {{site.SCALA_BINARY_VERSION}} 
+by default. (Spark can be built to work with other versions of Scala, too.) To write
```

Comment:
I don't follow. There are 2.10 and 2.11 artifacts in Maven for you to build against. There is actually only a 2.10 flavor of the release binaries. For simplicity I guess. No you can't mix/match scala versions.